lib/Twig/TemplateWrapper.php

```diff
@@ -47,6 +47,8 @@ public function render($context = array())
      * Displays the template.
      *
      * @param array $context An array of parameters to pass to the template
+     *
+     * @return string The rendered template
```

Comment:
Actually, I read too fast. `display` does not return anything. Instead, the `return` should be removed in the method code.